wsmaster/che-core-api-user/src/test/java/org/eclipse/che/api/user/server/UserManagerTest.java

```diff
@@ -45,13 +43,11 @@
     @Mock
     PreferenceDao  preferenceDao;
 
-    @InjectMocks
-    UserManager manager;
-
     @Test
     public void shouldCreateProfileAndPreferencesOnUserCreation() throws Exception {
         final User user = new User().withEmail("test@email.com").withName("testName");
-        manager.create(user, false);
+
+        new UserManager(userDao, profileDao, preferenceDao, new String[0]).create(user, false);
```

Comment:
mb it would be better to create instance of UserManager in method annotated with @BeforeMethod